cmake/kokkos_arch.cmake

```diff
@@ -17,6 +17,7 @@ ENDFUNCTION()
 KOKKOS_CFG_DEPENDS(ARCH COMPILER_ID)
 KOKKOS_CFG_DEPENDS(ARCH DEVICES)
 KOKKOS_CFG_DEPENDS(ARCH OPTIONS)
+#KOKKOS_CFG_DEPENDS(ARCH TPLS)
```

Comment:
What is this?

Reply:
I will drop.  That is the reason `KOKKOS_ENABLE_LIBQUADMATH` is not defined and I have to use `Kokkos_ENABLE_LIBQUADMATH`.  See the comment below.